scripts/build/termux_step_start_build.sh

```diff
@@ -4,6 +4,23 @@ termux_step_start_build() {
 	# Path to hostbuild marker, for use if package has hostbuild step
 	TERMUX_HOSTBUILD_MARKER="$TERMUX_PKG_HOSTBUILD_DIR/TERMUX_BUILT_FOR_$TERMUX_PKG_VERSION"
 
+	# Check if versions are aligned between linked packages.
+	# Also check if there are other packages wanting to keep version aligned with this package.
+	local PACKAGES_WANTING_VERSIONS_ALIGNED="$(grep -R -l -E 'TERMUX_PKG_ALIGN_VERSION_WITH=["'"'"']?'"$TERMUX_PKG_NAME"'["'"'"']?' --include="build.sh" ${TERMUX_SCRIPTDIR} ||:)"
+	if [ -n "${TERMUX_PKG_ALIGN_VERSION_WITH}" ]; then
```

Comment:
We should probably standardize this script to `[[  ]]` while we're here.
I also think this condition reads nicer as a bracegroup.
```suggestion
	[[ -n "${TERMUX_PKG_ALIGN_VERSION_WITH}" ]] && {
```

Reply:
It is `if [ ... ]; then` in all other conditions so I kept the coding style. Probably we should standardize this in separate refactoring PR.